public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -36,11 +31,7 @@ class AtomEdit extends React.Component {
     this.props.atomActions.updateAtom(newAtom);
   }
 
-  render () {
-
-    if (!this.props.atom) {
-      return <div>Loading...</div>;
-    }
+  renderSpecificEditor () {
 
     //TODO: This is brittle, can we improve?
     const atomType = this.props.atom.atomType.toLowerCase();
```

Comment:
I think we can get this from the key inside `data` now. Not great but at least it's the same case